### ISSUE TITLE
Issue upgrading to androidX

### ISSUE THREAD

**Issue by soumyamishra89** [Instabug/Instabug-React-Native#235]
instabug-reactnative version: 8.0.20
react-native version: 0.57.8

When enabling android.useAndroidX=true
android.enableJetifier=true I am getting the following error. 

```
Task :instabug-reactnative:compileReleaseJavaWithJavac FAILED
/client/node_modules/instabug-reactnative/android/src/main/java/com/instabug/reactlibrary/RNInstabugReactnativeModule.java:7: error: package android.support.annotation does not exist
import android.support.annotation.Nullable;
                                 ^
/client/node_modules/instabug-reactnative/android/src/main/java/com/instabug/reactlibrary/RNInstabugReactnativeModule.java:2161: error: cannot find symbol
                           @Nullable WritableMap params) {
                            ^
  symbol:   class Nullable
  location: class RNInstabugReactnativeModule
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
2 errors


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':instabug-reactnative:compileReleaseJavaWithJavac'.
> Compilation failed; see the compiler error output for details.
```

The issue is same as #231 but that is closed and no one has posted a solution.

Can someone help me with this?

**Comment by HeshamMegid** [comment]
@soumyamishra89 we're looking into it and will get back to you soon.

**Comment by salmamali** [comment]
@soumyamishra89 This is now fixed in v8.0.21. Please update to the latest.
